StPetersburg/InEtn66217.xml

```diff
@@ -0,0 +1,231 @@
+                  in <date>1969</date>.</acquisition>
+               </history>
+               <additional>
+                  <adminInfo>
+                     <recordHist>
+                        <source>
+                           <listBibl type="catalogue">
+                                 <bibl>
+                                    <ptr target="bm:Platonov1996Katalog"/>
+                                    <citedRange unit="page">65-66</citedRange>
+                                 </bibl>
+                           </listBibl>
+                              <listBibl type="secondary">
+                                 <bibl>
+                                    <ptr target="bm:Platonov2017RukKniga"/>
+                                      <citedRange unit="page">198</citedRange>
+                                 </bibl>
+                              </listBibl>
+                        </source>
+                     </recordHist>
+                  </adminInfo>
+
+               </additional>
+
+            </msDesc>
+
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <projectDesc>
+            <p>Encoded according to TEI P5 Guidelines.</p>
+         </projectDesc>
+         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude"
+                     href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+            <xi:fallback>
+               <p>Definitions of prefixes used.</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc>
+         <textClass>
+            <keywords>
+              <term key="ChristianLiterature"/>
+              <term key="Chronography"/>
+              <term key="Chants"></term>
+              <term key="Hagiography"></term>
+              <term key="AmharicLiterature"></term>
+              <term key="Poetry"></term>
+              <term key="Liturgy"></term>
+            </keywords>
+         </textClass>
+         <langUsage>
+           <language ident="en">English</language>
+           <language ident="gez">Gǝʿǝz</language>
+           <language ident="am">Amharic</language>
+            </langUsage>
+      </profileDesc>
+      <revisionDesc>
+         <change who="DE" when="2020-03-04">Created entry, added physical description and content.</change>
+      </revisionDesc>
+  </teiHeader>
+  <text xml:base="https://betamasaheft.eu/">
+      <body>
+        <div type="edition" xml:lang="gez">
+          <div type="textpart" subtype="supplication" xml:id="SupplicationMS" xml:lang="gez">
+            <ab>
+            ኦአቡየ፡ ቀሲስ፡ <space></space> ኢትርሥዓኒ፡ በጸሎትከ፡ ለእኁከ፡ <persName ref="PRS12883WaldaQ" role="owner">ወልደ፡ ቂርቆስ</persName>።
+            እንዘ፡ ትብል፡ ዕቀቦ፡ ወአድኅኖ፡ እምኩይ፡ ወእምኵሉ፡ ወንሱት፡ ለገበርከ፡ <persName ref="PRS12883WaldaQ" role="owner">ወልደ፡ ቂርቆስ፡ ጌታ፡ ሁን፡</persName>
```

Comment:
if እምኩይ and ለገበርከ are given like this in the catalogue, you could mark them up with sic to avoid confusion

Reply:
done